src/backend/egl/context.rs

```diff
@@ -37,6 +40,36 @@ pub struct EGLContext {
 unsafe impl Send for EGLContext {}
 
 impl EGLContext {
+    /// Creates a new `EGLContext` from raw handles.
+    ///
+    /// # Safety
+    ///
+    /// This function is marked unsafe, because the context cannot be made current on another thread without
+    /// being unbound again (see [`EGLContext::unbind`]).
+    pub unsafe fn from_raw<L>(
+        display: *const c_void,
+        config_id: *const c_void,
+        context: *const c_void,
```

Comment:
Guard against null pointers if applicable.

Reply:
idk if it is applicable, you can put a null pointer in but if you do that is your responsibility since it's an unsafe function?

Reply:
Well we can assert a pointer is not null at least.

Reply:
sure, should I make a new error type for that?

Reply:
Panicking is fine via the assert macros